packages/yew-macro/tests/function_component_attr/hook_location-fail.stderr

```diff
@@ -0,0 +1,41 @@
+error: hooks cannot be called at this position.
+ --> tests/function_component_attr/hook_location-fail.rs:9:9
+  |
+9 |         use_context::<Ctx>().unwrap();
+  |         ^^^^^^^^^^^
+
```

Comment:
There should be a "help" explaining why it can't be called here. `emit_error!` can do this and how to fix it. See [docs](https://docs.rs/proc-macro-error/latest/proc_macro_error/#note-attachments)